blocks/handler.go

```diff
@@ -71,7 +71,8 @@ func (h *Handler) HandleBlockData(ctx context.Context, data []byte) error {
 		logger.Info("known block")
 		return nil
 	}
-	logger.With().Info("new block", log.Inline(&b))
+	logger.With().Info("new block ID", log.String("block_id", b.ID().AsHash32().ShortString()))
```

Comment:
please keep the log as "new block/ballot/proposal/". the goal is logging an mesh event, not describing what we are logging.